task/label-check/0.1/label-check.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
   name: label-check
 spec:
   description: >-
-    Checks that image carries best practice labels.
+    Utilizes Conftest to check if image has best practice labels.
```

Comment:
Suggestion: Uses Conftest to verify whether an image contains the best practice labels, for example...?

Question:
1. Do we need to add some context to Conftest?